app-dashboard/js/view/panel-dashboard.js

```diff
@@ -48,13 +48,21 @@ define([
             let flood_acquisition_date = new Date(floodCollectionView.selected_forecast.attributes.acquisition_date);
             let flood_forecast_date = new Date(floodCollectionView.selected_forecast.attributes.forecast_date);
 
+            let lead_time = flood_forecast_date.getDate() - new Date().getDate();
```

Comment:
There is a better way to do this.
Each forecast will always have lead time information. So it's better to put extra attributes (lead_time) in forecast_event model.
Then this view here just fetches the data.

Reply:
use
```
let lead_time = floodCollectionView.selected_forecast.attributes.lead_time
```